src/lib/Microsoft.Health.Events/Telemetry/Exceptions/EventHubErrorCode.cs

```diff
@@ -8,19 +8,19 @@ namespace Microsoft.Health.Events.Telemetry
     public enum EventHubErrorCode
     {
         /// <summary>
-        /// Error code that categorizes exceptions of the type EventHubsException
+        /// Error code for an issue relating to an Event Hub (e.g. invalid name or consumer group)
         /// </summary>
-        OperationError,
+        InstanceError,
 
         /// <summary>
-        /// Error code that indicates failures in initializing event hub partition
+        /// Error code for an issue relating to an Event Hubs Namespace (e.g. invalid FQDN)
         /// </summary>
-        EventHubPartitionInitFailed,
+        NamespaceError,
```

Comment:
I would prefer combining these into a single "InvalidConfigError" or something like that to generally indicate that name/consumergroup/namespace is incorrect. Otherwise we will be exposing a shoebox metric for each configuration. 
But happy to hear your thoughts on this.

Reply:
As discussed offline, only 1 metric (TotalErrors) is exposed in shoebox and the user can filter on the "Name" dimension to see that there's an event hub configuration error. A specific error for each event hub configuration error is unneeded.
Therefore, I've combined `NamespaceError` and `InstanceError` into a single `ConfigurationError` (naming matches that of the other error codes; feel free to suggest another one).